VeriStand/Data Sharing Custom Device/Source/Data Sharing Framework Custom Device.lvproj

```diff
@@ -21,7 +21,16 @@
 		<Property Name="server.vi.callsEnabled" Type="Bool">true</Property>
 		<Property Name="server.vi.propertiesEnabled" Type="Bool">true</Property>
 		<Property Name="specify.custom.address" Type="Bool">false</Property>
+		<Item Name="Documents" Type="Folder">
+			<Item Name="DSF-errors.txt" Type="Document" URL="../../../../Docs/DSF-errors.txt"/>
+		</Item>
 		<Item Name="Tests" Type="Folder">
+			<Item Name="Scripting" Type="Folder">
+				<Item Name="Examples" Type="Folder">
+					<Item Name="Scripting.lvclass" Type="LVClass" URL="../Tests/Scripting/Add Loopback/Scripting.lvclass"/>
```

Comment:
Why is `Scripting.lvclass` in an **Add Loopback** folder on disk? This seems like it should just be under the **Scripting** directory. Same for `targets.ini`.

Reply:
originally going to add two test classes 'Add Loopback' and 'Import Send' but decided to include both tests in the same test class.

Reply:
https://github.com/ni/niveristand-data-sharing-framework-custom-device/pull/20/commits/023029302fd93e9682001b411fe35f7509b4402f